### DISCUSSION TITLE
Multiple Cascading Selects

### DISCUSSION THREAD

**Discussion by jefffdelima** [bigskysoftware/htmx#2253] [Q&A]
Hi Guy!

I am new to programming, and I found HTMX very interesting, useful and easy to learn.

I am just wondering if there is a workaround of imitating this one [https://www.w3schools.com/howto/tryit.asp?filename=tryhow_js_cascading_dropdown](https://www.w3schools.com/howto/tryit.asp?filename=tryhow_js_cascading_dropdown) using HTMX.

I already implemented the example of [Cascading Select](https://htmx.org/examples/value-select/) we have here on my project, but only works on 2 dropdowns. I wanted it that when the first dropdown was changed, all the others will be back to default settings.

Hope to hear some of your insights on this one.

Thanks, Guys! :)

HTMX Rocks!!!

**Comment by hdrk2k** [comment]
https://youtu.be/BxxDQOo_rNI?si=2HQK0pbOkT5Q-IiP
